src/components/Sidebar/NestedTableOfContentsSidebar.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import classnames from 'classnames';
+
+import Sidebar from './Sidebar';
+import { SidebarSection } from './Section';
+import { Parser } from "html-to-react"
+
+const NestedTableOfContentsSidebar = ({ tableOfContents, className }) => {
+  const htmlToReactParser = new Parser();
+
+  const reactComponents = htmlToReactParser.parse(tableOfContents);
+  return (
+    <Sidebar
+      side="right"
+      sticky={true}
+      className={classnames('hidden xl:block md:w-96 pr-3', className)}
+    >
+      <SidebarSection>
+        <strong>Table of Contents</strong>
+      </SidebarSection>
+
+      <SidebarSection>
+      {reactComponents}
```

Comment:
I would like to apply styles to this component as follows.

Please if you know how to do this, let me know.

```
ul li ul li {
  text-indent: 1em;
}

ul li ul li ul li {
  text-indent: 2em;
}

ul li ul li ul li ul li {
  text-indent: 3em;
}